src/app/support/donationBox.tsx

```diff
@@ -32,19 +37,22 @@ function DonationBox() {
   //obviously, we'll set this only we actually get the donation.
   //We'll have to write the logic when we get the info sent back from Helcim upon successful donation
   const [currentAmount, setCurrentAmount] = useState(0);
-
   const [formData, setFormData] = useState({
     donationCategory: "individual",
-    name: "",
+    firstName: "",
+    lastName: "",
     email: "",
     address: "",
     businessName: "",
+    contactName: "",
     selectedAmount: "",
     customAmount: "",
   });
-
   const [showAddress, setShowAddress] = useState(false);
+  const isDonationCategory = (category: string): category is DonationCategory =>
```

Comment:
I can tell this is some chatGPT mess... haha. I see what you were doing here though. You can delete this and I'll explain below where you used it.